src/harness/sas.py

```diff
@@ -76,6 +75,40 @@ def _Request(url, request, ssl_cert, ssl_key):
   logging.debug('Response:\n' + body)
   return json.loads(body)
 
+def _RequestGet(url, ssl_cert, ssl_key):
```

Comment:
There is no conn.setopt(conn.POST, True) in the GET Request. If this is not set to true then by default it will make a GET Request. Also, GET Request doesn't have any body in it, I also removed the conn.setopt(conn.POSTFIELDS, request) line.